sw/ground_segment/misc/natnet2ivy.c

```diff
@@ -19,14 +19,14 @@
  * Boston, MA 02111-1307, USA.
  */
 
-/** \file natnet2ivy.c
-*  \brief NatNet (GPS) to ivy forwarder
-*
-*   This receives aircraft position information through the Optitrack system
-* NatNet UDP stream and forwards it to the ivy bus. An aircraft with the gps
-* subsystem "datalink" is then able to parse the GPS position and use it to
-* navigate inside the Optitrack system.
-*/
+ /** \file natnet2ivy.c
+ *  \brief NatNet (GPS) to ivy forwarder
+ *
+ *   This receives aircraft position information through the Optitrack system
+ * NatNet UDP stream and forwards it to the ivy bus. An aircraft with the gps
+ * subsystem "datalink" is then able to parse the GPS position and use it to
+ * navigate inside the Optitrack system.
+ */
 
```

Comment:
What are exactly the changes in this file beside the logging and breaking a part of the code style ? Is this a duplicate of #1538 ?